### DISCUSSION TITLE
Translate Entry Relationships

### DISCUSSION THREAD

**Discussion by aloco** [craftcms/cms#11030] [Q&A] [answered]
Hi,

I am not sure if I am overlooking something here. 

I have a Multisite with two languages, DE and EN. The following settings are applied for an entry relationship field
<img width="857" alt="Screenshot 2022-04-27 at 18 32 35" src="https://user-images.githubusercontent.com/11661125/165572229-dafa8bb7-d8ba-43c8-924d-3147304112d1.png">

I am using this field in globals. In the entry picker I can switch between entries from the EN site and the DE site. I would like to link entries from DE on the EN page, because the content is not available in both languages.

The entry field looks like this after selecting an entry from DE on the current EN global field
 
<img width="486" alt="Screenshot 2022-04-27 at 18 31 13" src="https://user-images.githubusercontent.com/11661125/165574295-cdfca7cb-e6e3-4fa6-b10c-872a24ae7f6f.png">

After saving the globals, it becomes this: 
<img width="484" alt="Screenshot 2022-04-27 at 18 30 35" src="https://user-images.githubusercontent.com/11661125/165574387-e89acb53-149d-4498-a1f2-86ec44681bab.png">
it looks like it is using the EN version which is disabled because there is no content yet.

Is this the expected behaviour? If yes I really don´t understand the setting "Manage relations on a per-site basis"

Thank you for clarifying this

**Comment by brandonkelly** [comment, marked as answer]
Yes, this is expected. Craft does not store a target site for each relationship individually.

On the front-end, you can fetch the related entries across other sites besides the current one if necessary, like so:

```twig
{% set relatedEntries = entry.bestOfProjects
    .site('*')
    .preferSites([entry.siteId])
    .unique()
    .all() %}
```
